english/tos/tos.md

```diff
@@ -0,0 +1 @@
+[Tree of Savior](https://treeofsavior.com/) is the spiritual successor to the classic MMORPG [Ragnarok Online](iro.ragnarokonline.com/)
```

Comment:
This line contains more than one link.  Please put links on separate lines with a blank line in between so that options will be visually separated.